api/gateway.proto

```diff
@@ -125,8 +125,12 @@ message Gateway {
   DownlinkPathConstraint downlink_path_constraint = 18 [(validate.rules).enum.defined_only = true];
   // Adjust the time that GS schedules class C messages in advance. This is useful for gateways that have a known high latency backhaul, like 3G and satellite.
   google.protobuf.Duration schedule_anytime_delay = 19 [(gogoproto.stdduration) = true, (gogoproto.nullable) = true];
+  // update the location of this gateway from status messages
+  bool update_location_from_status = 21;
+  // debounce time for gateway location updates from status messages.
+  google.protobuf.Duration update_location_from_status_debounce_time = 22 [(gogoproto.stdduration) = true, (gogoproto.nullable) = true];
```

Comment:
I'm not sure how smart it is to let users configure this. The entire idea of debouncing is that we protect our database from malicious users/gateways, so if those users can just disable that protection by setting it to one nanosecond, we can just as well not debounce at all.

Reply:
Changed to a global GS option, defaulting to 1 hour per gateway.